src/main/protobuf/v1/relationship.proto

```diff
@@ -51,7 +51,8 @@ message InstitutionUpdateV1 {
   optional string description = 2;
   optional string digitalAddress = 3;
   optional string address = 4;
-  optional string taxCode = 5;
+  optional string zipCode = 5;
+  optional string taxCode = 6;
```

Comment:
If you are going to wipe out all the persisted data, there is no problem, otherwise, be careful here.

Reply:
yes, we are going to wipe all the data